JASP-Engine/JASP/R/graphPackage.R

```diff
@@ -0,0 +1,1088 @@
+.graphOptions <- list2env(list(
```

Comment:
You should not include this file in the commit, since the R package should be put in the JASP folder on your computer.